src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Internal/MsQuicAddressHelpers.cs

```diff
@@ -32,29 +33,11 @@ internal static SOCKADDR_INET IPEndPointToINet(IPEndPoint endpoint)
                 switch (endpoint.Address.AddressFamily)
                 {
                     case AddressFamily.InterNetwork:
-                        socketAddress.Ipv4.sin_addr0 = buffer[0];
-                        socketAddress.Ipv4.sin_addr1 = buffer[1];
-                        socketAddress.Ipv4.sin_addr2 = buffer[2];
-                        socketAddress.Ipv4.sin_addr3 = buffer[3];
+                        endpoint.Address.TryWriteBytes(MemoryMarshal.CreateSpan<byte>(ref socketAddress.Ipv4.sin_addr, 4));
```

Comment:
Please fix the compilation failures.

Reply:
Done. All CI tests now run successfully.